modules/RTC/RTCUtils.js

```diff
@@ -321,6 +321,7 @@ function pollForAvailableMediaDevices() {
  * @emits RTCEvents.DEVICE_LIST_CHANGED
  */
 function onMediaDevicesListChanged(devices) {
+    var initialDeviceList = currentlyAvailableMediaDevices.length == 0;
```

Comment:
We can fire one more event, e.g. RTCEvents.DEVICE_LIST_AVAILABLE, and log audioOutput device on that event:
```javascript
function onReady (options, GUM) {
    rtcReady = true;
    eventEmitter.emit(RTCEvents.RTC_READY, true);
    screenObtainer.init(options, GUM);

    if (RTCUtils.isDeviceListAvailable() && rawEnumerateDevicesWithCallback) {
        rawEnumerateDevicesWithCallback(function (devices) {
            currentlyAvailableMediaDevices = devices.splice(0);
            
            eventEmitter.emit(RTCEvents.DEVICE_LIST_AVAILABLE, currentlyAvailableMediaDevices);

            if (isDeviceChangeEventSupported) {
                navigator.mediaDevices.addEventListener('devicechange', function () {
                    RTCUtils.enumerateDevices(onMediaDevicesListChanged);
                });
            } else {
                window.setTimeout(pollForAvailableMediaDevices,
                    AVAILABLE_DEVICES_POLL_INTERVAL_TIME);
            }
        });
    }
}
```

And this way we will get the initial device list and also won't need to re-do anything after switching to native "devicechange" event.